src/adhocracy_pcompass/adhocracy_pcompass/resources/subscriber.py

```diff
@@ -32,31 +35,27 @@ def notify_policycompass(event):
 
     # this is not a known policycompass external resource
     if match is None:  # pragma: no cover
-        return
+        return ''
```

Comment:
I would rather through an ValueError exception here no? Because it's used in `requests.post` after, what would be the effect if the argument is an empty string?